src/ucs/memory/rcache.c

```diff
@@ -830,11 +830,7 @@ ucs_rcache_check_overlap_one(ucs_rcache_t *rcache, ucs_pgt_addr_t *start,
     return UCS_OK;
 }
 
-/* Lock must be held
- * Old ASAN versions reports buffer underflow false-positive during access to
- * `region` through `ucs_list_link_t` entry. Newer ASAN versions don't
- * report this issue (e.g. standard ASAN on Ubuntu 22.04).
- */
+/* Lock must be held */
 static ucs_status_t UCS_F_NO_SANITIZE_ADDRESS
```

Comment:
If there is no leak anymore then we can remove `UCS_F_NO_SANITIZE_ADDRESS`

```suggestion
static ucs_status_t
```